### ISSUE TITLE
Add bitmap font detection function

### ISSUE THREAD

**Issue by ftphikari** [libsdl-org/SDL_ttf#169]
Freetype supports COLR/CPAL/SVG (vector based) as well as CBDT/CBLC (bitmap based png's) fonts. However I cannot find a function in SDL_ttf API that can tell me if the font is bitmap or not. If it doesn't exist yet - it should. Am I missing something?

**Comment by 1bsyl** [comment]
But I think the fonts can have both vector and png at some various size.
what about those functions ?


```c
/* Get the number of faces of the font */
extern DECLSPEC long SDLCALL TTF_FontFaces(const TTF_Font *font);

/* Get the font face attributes, if any */
extern DECLSPEC int SDLCALL TTF_FontFaceIsFixedWidth(const TTF_Font *font);
extern DECLSPEC char * SDLCALL TTF_FontFaceFamilyName(const TTF_Font *font);
extern DECLSPEC char * SDLCALL TTF_FontFaceStyleName(const TTF_Font *font);
```

https://github.com/libsdl-org/SDL_ttf/blob/main/SDL_ttf.h#L179

**Comment by ftphikari** [comment]
> But I think the fonts can have both vector and png at some various size.

Then it would be nice if i could tell which glyph was png and which was vector.

> what about those functions ?

Not sure what you mean here, how can i use them to detect what i want?

**Comment by 1bsyl** [comment]
Open the font 
maybe if TTF_FontFaceIsFixedWidth() is true then this is PNG (or something else fixed?) otherwise vector

**Comment by ftphikari** [comment]
> maybe if TTF_FontFaceIsFixedWidth() is true then this is PNG (or something else fixed?) otherwise vector

Yeah, maybe it is a png, or maybe it is a monospaced font. So it doesn't help.

**Comment by 1bsyl** [comment]
yes, true, I am wrong. PNB would be not scalable in fact. (should use "`FT_IS_SCALABLE(face)`")
Maybe it's not per glyph, but per size / face. Not sure, maybe this can be fixed.

maybe you can submit a PR so that you can also check how it works
